Alcatraz/Alcatraz-Info.plist

```diff
@@ -24,6 +24,7 @@
 	<string>1</string>
 	<key>DVTPlugInCompatibilityUUIDs</key>
 	<array>
+		<string>992275C1-432A-4CF7-B659-D84ED6D42D3F</string>
```

Comment:
@jurre @kattrali nitpick: are we adding new UUIDs on the bottom or beginning of the array?